src/main/java/io/strimzi/kafka/bridge/Application.java

```diff
@@ -51,6 +55,16 @@ public void start() {
                 log.debug("Failed to deploy verticle instance", done.cause());
             }
         });
+
+        this.vertx.deployVerticle(this.httpBridge, done -> {
+
+            if (done.succeeded()) {
+                log.debug("Verticle instance deployed [{}]", done.result());
```

Comment:
The log doesn't refer to the HTTP bridge instance. It should be clear. At same time the log on deploying the AMQP verticle should be changed as well. At same time at the bottom on `stop` we should just say "Kafka bridge" and not "AMQP-Kafka bridge"